### PR TITLE
update CP2K easyblock w.r.t. running regtest for CP2K v8.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="CP2K-5.1-intel-2020a.eb CP2K-6.1-intel-2020a.eb CP2K-7.1-foss-2020b.eb CP2K-7.1-intel-2020b.eb"
CORE_CNT=16

File: easybuild/easyblocks/c/cp2k.py [hunk 1]
```diff
@@ -701,13 +701,20 @@ def test_step(self):
...
 
             # location of do_regtest script
-            cfg_fn = "cp2k_regtest.cfg"
+            cfg_fn = 'cp2k_regtest.cfg'
+
             regtest_script = os.path.join(self.cfg['start_dir'], 'tools', 'regtesting', 'do_regtest')
-            regtest_cmd = "%s -nosvn -nobuild -config %s" % (regtest_script, cfg_fn)
+            regtest_cmd = [regtest_script, '-nobuild', '-config', cfg_fn]
+            if LooseVersion(self.version) < LooseVersion('8.0'):
+                # -nosvn option was removed in CP2K 8.1
```
Comment (smoors) [inline easybuild/easyblocks/c/cp2k.py:709]: it was already removed in 7.1, but just shows a warning there, so should be fine

Comment (boegel) [reply easybuild/easyblocks/c/cp2k.py:709]: It was deprecated in 7.1, yeah, but that doesn't cause any actual trouble...

Should I change the version check to `< 7.0`?

Comment (smoors) [reply easybuild/easyblocks/c/cp2k.py:709]: you could, or you could change the comment to say '-nosvn is not allowed in CP2K 8.1' or something like that.
both are fine to me

Comment (boegel) [reply easybuild/easyblocks/c/cp2k.py]: fixed in 31f917b

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _CP2K-7.1-foss-2020b.eb_ 
 * **FAIL (build issue)** _CP2K-7.1-intel-2020a.eb_ (partial log available at https://gist.github.com/1839d52712d4d16d85069c5e50ddb759)

Build succeeded for 1 out of 2 (2 easyconfigs in total)
node3113.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/7545bbc7cebd827a0e2953f8cdec81b1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _CP2K-7.1-foss-2020b.eb_ 
 * **SUCCESS** _CP2K-7.1-intel-2020a.eb_ 

Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3113.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/84662ed1867bc114c29f0e429a867562 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**